test/integration/assessment/remove-assessment.ts

```diff
@@ -0,0 +1,24 @@
+import { AssessmentService } from '@server/service/assessment'
+
+export default () =>
+  test('Expect assessment to be removed', async () => {
+    const assessmentParams = {
+      props: {
+        name: 'fra',
+        cycles: ['1'],
+      },
+    }
+
+    const assessment = await AssessmentService.read({
+      assessment: assessmentParams,
+    })
+
+    const removedAssessment = await AssessmentService.remove({
+      assessment,
+    })
+
```

Comment:
here better to read again the assessment and verify you get an exception that the there's no assessment with that name